R/prep_data.R

```diff
@@ -158,8 +182,8 @@ prep_data <- function(d,
   }
   if (no_prep)
     remove_near_zero_variance <- convert_dates <- impute <-
-      collapse_rare_factors <- center <- scale <- make_dummies <-
-      add_levels <- logical_to_numeric <- factor_outcome <- FALSE
+    collapse_rare_factors <- center <- scale <- make_dummies <-
+    add_levels <- logical_to_numeric <- factor_outcome <- FALSE
```

Comment:
I think you do need these indents, as they are a continuation of the previous line.